.github/actions/shared-node-cache/action.yml

```diff
@@ -0,0 +1,28 @@
+name: 'Shared Node Cache'
```

Comment:
This is based on the action that @jaredly wrote for https://github.com/Khan/webapp.